src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptor.java

```diff
@@ -418,7 +418,7 @@ private interface Rejector {
                 Checker.preCheckedCast(field.getType(), value, false, false, false);
                 return super.onSetAttribute(invoker, receiver, attribute, value);
             } else {
-                throw StaticWhitelist.rejectField(field);
+                rejector = () -> StaticWhitelist.rejectField(field);
```

Comment:
Not related to JENKINS-69899, but fixes a minor issue in https://github.com/jenkinsci/script-security-plugin/commit/85d16bd851b355458f6396dbc4f92c3e0eebcd72. See https://github.com/jenkinsci/script-security-plugin/pull/298#discussion_r999655554 for details.

I am fixing it here in case we end up backporting the JENKINS-69899 fix, since it would be nice to backport this as well. That said, the cases it fixes were already broken before https://github.com/jenkinsci/script-security-plugin/commit/85d16bd851b355458f6396dbc4f92c3e0eebcd72, so it is not particularly important).